technique/reflectometry/base.py

```diff
@@ -255,6 +255,59 @@ def slit_check(theta, footprint, resolution):
     print("s2vg={}".format(s2))
 
 
+def auto_height(laser_block: str, fine_height_block: str, target: float = 0.0, continue_if_NaN: bool=False):
+    """
+    Moves the sample fine height axis so that it is centred on the beam, based on the readout of a laser height gun.
```

Comment:
In run_angle there is a call to an auto_height function, I believe this should probably be calling this function with the correct blocks?